src/gitingest/ingest_from_query.py

```diff
@@ -1,6 +1,7 @@
 import os
 from fnmatch import fnmatch
 from typing import Any
+from PyPDF2 import PdfReader
```

Comment:
Maybe I'm missing something but why do we need `PyPDF2` when the official docs mentions `pypdf`

![image](https://github.com/user-attachments/assets/3c158b3b-d162-4cc6-ba09-e13886597247)

Reply:
Made the necessary changes to use the latest documentation.